release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,7 +6,11 @@ uid: DCP_change_log
 
 The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
-#### 12 June 2023 - Fix - CloudGateway 2.11.0 / CoreGateway 2.13.0 - Resolved connection issue [ID_36439] [ID_36453]
+#### 19 June 2023 - Fix - Chat Integration with Microsoft Teams: Improved consistency for email inputs [ID_36643]
+
+Consistency for the Chat Integration email inputs was improved because some features were using the email input as the actual email address of the user, and some as the service principal name of the user. This is not an issue if, in your linked tenant, both have the same value. If the values would differ, some features like 'Create Team' & 'Add Team Member or Owner' would not work unless the actual service principl name was given. Since today this is no longer the case and all email inputs for all Chat Integration features behave the same as the email address set for the user in your tenant.
+
+#### 7 June 2023 - Fix - CloudGateway 2.11.0 / CoreGateway 2.13.0 - Resolved connection issue [ID_36439] [ID_36453]
```

Comment:
I corrected the date to the DxM publish date instead of the date when this change log was created